### PR TITLE
test: Reduce test image size

### PR DESCRIPTION
**Pull request by dperpeet**

### CODE REVIEW

**Comment by dperpeet** [comment]
I'm not sure it's worth the hack: [a fedora 23 image from Jan 29th](https://fedorapeople.org/groups/cockpit/images/fedora-23-0cb312e9fcbd50c42e026f6bfc7ebcd802c11be3.qcow2.xz) is 1880171712 bytes large when compressed, while the new one has 1799504424 bytes (roughly a 5% decrease). This at the cost of a hack and filling the virtual disk once (if the image weren't sparse, it'd be larger).

**Comment by dperpeet** [comment]
By changing qemu-img parameters to compress our clean images, we greatly reduce the file size.

**Comment by dperpeet** [comment]
With the latest changes, images are significantly smaller: we use the qcow2 internal compression. When building and installing packages with the compressed image as a backing store, the read-write layer is uncompressed. When testing this locally, vm-install wasn't slower with compressed images than with the uncompressed images. Test-runners will show how quickly the actual test suites run.

**Comment by dperpeet** [comment]
some results:
old images

- fedora 23:
  # TESTS PASSED [1214s on files]
- fedora-atomic:
# TESTS PASSED [638s on host32-rack06]

new images
- fedora 23:
# TESTS PASSED [1101s on host37-rack09]
- fedora-atomic:
# TESTS PASSED [712s on host32-rack06]

**Comment by dperpeet** [comment]
Changes break Debian image because I deleted `/var/cache/pbuilder`

**Comment by dperpeet** [comment]
issue is fixed, waiting for image upload debian-unstable to push

File: test/guest/lib/atomic.setup [hunk 1]
```diff
@@ -26,15 +26,11 @@ atomic host upgrade
...
+/var/lib/testvm/docker-images.setup
 
-# pull cockpit/ws so we don't need to download this for every test
-docker pull cockpit/ws
-docker pull busybox:latest
-docker pull busybox:buildroot-2014.02
-docker pull submod/helloapache:0.1.11
-docker pull gcr.io/google_containers/pause:0.8.0
+# reduce image size
+/var/lib/testvm/zero-disk.setup
```
Comment (mvollmer) [inline test/guest/lib/atomic.setup]: Should this be at the end of the script?

**Pull request closed without merging**